src/main/webapp/app/course/manage/course-management.service.ts

```diff
@@ -160,7 +160,7 @@ export class CourseManagementService {
     }
 
     /**
-     * finds one course using a GET request
+     * finds one course using a GET request. If the course was already loaded it can be retrieved using {@link CourseStorageService#getCourse} or {@link CourseStorageService#subscribeToCourseUpdates}
```

Comment:
For dev-builds, you could log when a developer calls this method although the course is already loaded to indicate redundant REST calls.

Reply:
Good idea. Done!